### PR TITLE
Add Quality Declarations and READMEs for each package

### PR DESCRIPTION
**Pull request by brawner**
Adding current-level Quality Declarations and READMEs for all three packages in this repo.

Signed-off-by: Stephen Brawner <brawner@gmail.com>

### CODE REVIEW

File: fastrtps_cmake_module/QUALITY_DECLARATION.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+`fastrtps_cmake_module` does not provide any public API of its own.
+
+### API Stability Within a Released ROS Distribution [1.iv]/[1.vi]
+
+`fastrtps_cmake_module` does not provide its own API and therefore API stability will not be impacted by changes.
+
+### ABI Stability Within a Released ROS Distribution [1.v]/[1.vi]
+
+`fastrtps_cmake_module` does not contain any c or c++ code, therefore changes will not affect ABI stability.
```
Comment (ahcorde) [inline fastrtps_cmake_module/QUALITY_DECLARATION.md:29]:
```suggestion
`fastrtps_cmake_module` does not contain any C or C++ code, therefore changes will not affect ABI stability.
```

File: fastrtps_cmake_module/QUALITY_DECLARATION.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+This package uses DCO as its confirmation of contributor origin policy. More information can be found in [CONTRIBUTING](../CONTRIBUTING.md).
+
+### Peer Review Policy [2.iii]
+
+Following the recommended guidelines for ROS Core packages, all pull requests must have at least 1 peer review.
+
+### Continuous Integration [2.iv]
+
+All pull request must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers)
```
Comment (ahcorde) [inline fastrtps_cmake_module/QUALITY_DECLARATION.md:49]:
```suggestion
All pull request must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers).
```

File: rosidl_typesupport_fastrtps_c/QUALITY_DECLARATION.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+## Change Control Process [2]
+
+`rosidl_typesupport_fastrtps_c` follows the recommended guidelines for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#package-requirements).
+
+### Change Requests [2.i]
+
+This package requires that all changes occur through a pull request.
+
+### Contributor Origin [2.ii]
+This package uses DCO as its confirmation of contributor origin policy.
```
Comment (ahcorde) [inline rosidl_typesupport_fastrtps_c/QUALITY_DECLARATION.md:42]: new line

File: rosidl_typesupport_fastrtps_c/QUALITY_DECLARATION.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+More information can be found in [CONTRIBUTING](../CONTRIBUTING.md).
+
+### Peer Review Policy [2.iii]
+
+Following the recommended guidelines for ROS Core packages, all pull request have at least 1 peer review.
+
+### Continuous Integration [2.iv]
+
+All pull request must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers)
```
Comment (ahcorde) [inline rosidl_typesupport_fastrtps_c/QUALITY_DECLARATION.md:51]:
```suggestion
All pull request must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers).
```

File: rosidl_typesupport_fastrtps_cpp/QUALITY_DECLARATION.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+## Change Control Process [2]
+
+`rosidl_typesupport_fastrtps_cpp` follows the recommended guidelines for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#package-requirements).
+
+### Change Requests [2.i]
+
+This package requires that all changes occur through a pull request.
+
+### Contributor Origin [2.ii]
+This package uses DCO as its confirmation of contributor origin policy.
```
Comment (ahcorde) [inline rosidl_typesupport_fastrtps_cpp/QUALITY_DECLARATION.md:42]: new line

File: rosidl_typesupport_fastrtps_cpp/QUALITY_DECLARATION.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+More information can be found in [CONTRIBUTING](../CONTRIBUTING.md).
+
+### Peer Review Policy [2.iii]
+
+Following the recommended guidelines for ROS Core packages, all pull request have at least 1 peer review.
+
+### Continuous Integration [2.iv]
+
+All pull request must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers)
```
Comment (ahcorde) [inline rosidl_typesupport_fastrtps_cpp/QUALITY_DECLARATION.md:51]:
```suggestion
All pull request must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers).
```

File: fastrtps_cmake_module/QUALITY_DECLARATION.md (hunk 1 above)
Comment (brawner) [reply fastrtps_cmake_module/QUALITY_DECLARATION.md]: Done.

File: rosidl_typesupport_fastrtps_c/QUALITY_DECLARATION.md (hunk 4 above)
Comment (brawner) [reply rosidl_typesupport_fastrtps_c/QUALITY_DECLARATION.md]: Done.

File: rosidl_typesupport_fastrtps_cpp/QUALITY_DECLARATION.md (hunk 5 above)
Comment (brawner) [reply rosidl_typesupport_fastrtps_cpp/QUALITY_DECLARATION.md:42]: Added.

File: rosidl_typesupport_fastrtps_cpp/QUALITY_DECLARATION.md (hunk 6 above)
Comment (brawner) [reply rosidl_typesupport_fastrtps_cpp/QUALITY_DECLARATION.md]: Done.

File: fastrtps_cmake_module/QUALITY_DECLARATION.md (hunk 2 above)
Comment (brawner) [reply fastrtps_cmake_module/QUALITY_DECLARATION.md]: Fixed.

File: rosidl_typesupport_fastrtps_c/QUALITY_DECLARATION.md (hunk 3 above)
Comment (brawner) [reply rosidl_typesupport_fastrtps_c/QUALITY_DECLARATION.md:43]: Fixed.

File: fastrtps_cmake_module/QUALITY_DECLARATION.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+### License [3.iii]
+
+The license for `fastrtps_cmake_module` is Apache 2.0, the type is declared in the [package.xml](package.xml) manifest file, and a full copy of the license is in the [LICENSE](../LICENSE) file.
+
+There are no source files in this package and therefore linters do not check for a license statement.
+
+### Copyright Statements [3.iv]
+
+The copyright holders each provide a statement of copyright in each source code file in `rcpputils`.
```
Comment (ahcorde) [inline fastrtps_cmake_module/QUALITY_DECLARATION.md:73]:
```suggestion
The copyright holders each provide a statement of copyright in each source code file in `fastrtps_cmake_module`.
```

File: fastrtps_cmake_module/QUALITY_DECLARATION.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+
+The results of the test can be found [here](https://ci.ros2.org/view/nightly/job/nightly_linux_release/lastBuild/testReport/fastrtps_cmake_module/copyright/).
+
+## Testing [4]
+
+`fastrtps_cmake_module` is a package providing solely CMake files and therefore does not require tests and has no coverage or performance requirements.
+
+### Linters and Static Analysis [4.v]
+
+`rmw` uses and passes all the standard linters and static analysis tools for a CMake package as described in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#linters).
```
Comment (ahcorde) [inline fastrtps_cmake_module/QUALITY_DECLARATION.md:85]:
```suggestion
`fastrtps_cmake_module` uses and passes all the standard linters and static analysis tools for a CMake package as described in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#linters).
```

File: fastrtps_cmake_module/QUALITY_DECLARATION.md (hunk 7 above)
Comment (brawner) [reply fastrtps_cmake_module/QUALITY_DECLARATION.md]: Oh my! Fixed.

File: fastrtps_cmake_module/QUALITY_DECLARATION.md (hunk 8 above)
Comment (brawner) [reply fastrtps_cmake_module/QUALITY_DECLARATION.md]: Fixed.

**Pull request merged**